backend/danswer/db/document_set.py

```diff
@@ -519,6 +523,70 @@ def fetch_documents_for_document_set_paginated(
     return documents, documents[-1].id if documents else None
 
 
+def select_documents_by_docset(
```

Comment:
nit: prefer some prefix to indicate that this isn't a query and gives a statement instead